src/rpc/server.cpp

```diff
@@ -191,11 +220,37 @@ static UniValue uptime(const JSONRPCRequest& jsonRequest)
     return GetTime() - GetStartupTime();
 }
 
+static UniValue getrpcinfo(const JSONRPCRequest& request)
+{
+    if (request.fHelp || request.params.size() > 0) {
+        throw std::runtime_error(
+            RPCHelpMan{"getrpcinfo",
+                "\nReturns details of the RPC server.\n", {}}
+                .ToString()
```

Comment:
14982: add missing parts from 14987 we backported earlier
```suggestion
                "\nReturns details of the RPC server.\n",
                {},
                RPCResults{},
                RPCExamples{""},
            }.ToString()
```